### PR TITLE
#5 add some tests

### PR DESCRIPTION
**Pull request by Vichukano**

### CODE REVIEW

**Comment by yegor256** [comment]
@Vichukano many thanks, but I'm not sure it will pass our quality check. Try to run `mvn clean install -Pqulice` on your computer.

**Comment by Vichukano** [comment]
@yegor256 thanks! code style fixed

File: src/test/java/com/yegor256/xsline/StClasspathTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * Test case for {@link StLambda}.
+ *
+ * @since 0.4.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+class StClasspathTest {
+
+    @Test
+    void shouldThrowIfResourceIsAbsent() {
+        try {
```
Comment (yegor256) [inline src/test/java/com/yegor256/xsline/StClasspathTest.java:40]: @Vichukano try to use `Assertions.assertThrows` instead

File: src/test/java/com/yegor256/xsline/TrXslTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test case for {@link TrXSL}.
+ *
+ * @since 0.4.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+class TrXslTest {
```
Comment (yegor256) [inline src/test/java/com/yegor256/xsline/TrXslTest.java:39]: @Vichukano I think a better name for this class would be `TrXSLTest`

File: src/test/java/com/yegor256/xsline/TrXslTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import com.jcabi.xml.XSLDocument;
+import java.io.IOException;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test case for {@link TrXSL}.
+ *
+ * @since 0.4.0
```
Comment (yegor256) [inline src/test/java/com/yegor256/xsline/TrXslTest.java:36]: @Vichukano it's better to mention `0.6.0` here, since `0.5.0` has already been released

**Comment by yegor256** [comment]
@Vichukano looks much better now! See my comments above

File: src/test/java/com/yegor256/xsline/TrXslTest.java (hunk 2 above)
Comment (Vichukano) [reply src/test/java/com/yegor256/xsline/TrXslTest.java]: fix it

File: src/test/java/com/yegor256/xsline/TrXslTest.java (hunk 3 above)
Comment (Vichukano) [reply src/test/java/com/yegor256/xsline/TrXslTest.java]: fix it

File: src/test/java/com/yegor256/xsline/TrXslTest.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        final Train<Shift> train = new TrXSL<>(new TrDefault<>())
+            .with(xsl)
+            .back();
+        MatcherAssert.assertThat(train, Matchers.iterableWithSize(1));
+    }
+
+    @Test
+    void shouldThrow() throws IOException {
+        final XSL xsl = new XSLDocument(this.getClass().getResource("add-brackets.xsl"));
+        try {
```
[the remark this answers is not in the document]
Comment (Vichukano) [reply src/test/java/com/yegor256/xsline/TrXslTest.java]: fix it

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@Vichukano many thanks!

**Pull request merged**